app/screens/AppNavigator/screens/Balances/screens/SendConfirmationScreen.tsx

```diff
@@ -130,7 +130,7 @@ export function SendConfirmationScreen ({ route }: Props): JSX.Element {
       <SubmitButtonGroup
         isDisabled={isSubmitting || hasPendingJob || hasPendingBroadcastJob}
         label={
-          isSubmitting
+          isSubmitting || hasPendingJob || hasPendingBroadcastJob
```

Comment:
Do correct me if I am wrong, will there be a scenario where we have pending job from other transaction but we want to display a contextual loading message here? 

Yep, it would be useful if that job happen to be "Sending", but it can also be "Converting" or "Adding" no? In my opinion, I think by disabling the button is sufficient for this case

Similarly for all the other <SubmitButtonGroup> implementation

Reply:
Also, would it be more readable if we add a new prop that represent loading label explicitly? And another prop that indicate the loading state?

This is a question, it may be difficult to implement I don't know, happy to discuss

Reply:
In my opinion Loader should be visible only when `isSubmitting ` is true, But as we are dispatching txn into queue and just updating the state of `isSubmitting` to false, we are getting some delay in opening the confirmation screen and turning off Loader, Thats why I have added `isSubmitting || hasPendingJob || hasPendingBroadcastJob` items too for the loader, though As you mentioned if suppose we initiated the txn and went for second one then before confirmation screen we have one screen where we fill the values of txn (For eg SendComponent) there we are not allowing user to press Send button until previous txn is in progress (using same condition) https://github.com/DeFiCh/wallet/blob/main/app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx#L62. so user wont be able to visit confirm screen until old txn is not get submitted/confirmed
```
async function onSubmit (): Promise<void> {
    if (hasPendingJob || hasPendingBroadcastJob) {
      return
    }
```